tests/factories/cht/reports/pregnancy.js

```diff
@@ -15,6 +16,16 @@ const defaultSubmitter = {
   }
 };
 
+const edd = faker.date
+  .between({ from: '2024-06-01T00:00:00.000Z', to: '2024-06-30T00:00:00.000Z' })
+  .toISOString()
+  .split('T')[0];
+
+const lmp = faker.date
+  .between({ from: '2023-11-01T00:00:00.000Z', to: '2023-11-30T00:00:00.000Z' })
+  .toISOString()
+  .split('T')[0];
```

Comment:
```suggestion
const lmp = moment().subtract(3, 'months');
```
It can be challenging to determine the number of weeks pregnant based on a random date.